src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,25 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React from 'react';
+import styled from 'styled-components'
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import { VideoCard} from '../../components/Cards/Video/Video.component'
 
-function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+const Container = styled.section`
+  display: flex;
+  flex: 1 1 0%;
+  flex-flow: row wrap;
+  flex-grow: 0;
+  max-width: 100%;
+  flex-basis: 100%;
+  justify-content: flex-start;
+  margin: 10px;
+  width: 100%;
+`;
 
+function HomePage() {
   return (
-    <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
-    </section>
+    <Container>
+      <VideoCard />
+    </Container>
```

Comment:
Following my previous explanation, we would retrieve the data here and iterate over it:

```suggestion
    <Container>
      { data.items.map((item, index) => (
        <VideoCard
        key={ index }
        image={ item.snippet.thumbnails.medium.url }
        title={ item.snippet.title }
        description={ item.snippet.description }
        />
      ))
      }
   </Container>
```